### PR TITLE
Improve Readme.md file

### PR DESCRIPTION
**Pull request by RicardoBritoBrens**
Fix some typo and just make it more readable.

### CODE REVIEW

**Comment by evaldez89** [comment]
@RicardoBritoBrens pull recent changes from master.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,116 @@
...
+- [ ] Banco BDI :alarm_clock:
+- [ ] Banco López de Haro :alarm_clock:
+- [ ] Banco Ademi :alarm_clock:
+- [ ] Banco BELLBANK :alarm_clock:
+- [ ] Banco Múltiple Activo Dominicana :alarm_clock:
 
-## The current supported banks are:
-  - Banco BHD León (Dominican Republic)
-  - Banco Vimenca (Dominican Republic) `(in progress)`
+### Bancos Extranjeros
```
[the remark this answers is not in the document]
Comment (evaldez89) [reply README.md]: @RicardoBritoBrens FYI

**Review by evaldez89 — CHANGES_REQUESTED** [review]
Actually we wouln't want to predefined the banks the app will work with, we want to go further than our reach. In example, a person from another contry could add a bank we don't know of. That being said:

- [x] Just highligth available banks and banks in progress

**Comment by evaldez89** [comment]
@RicardoBritoBrens :robot:

**Comment by RicardoBritoBrens** [comment]
Changes done.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,47 @@
...
-  - Gmail: Using the Google api with readonly permission
+## Known Major Tasks TODO
 
+  1. When running the main file, one parameter should be the user email to determine which mail service to use
+  2. Right now it's only expecting multipart emails (threads) not single messages
+  3. How to handle email credentials?
+  4. How to handle email authentication token?
+  5. Add Outlook Email service
+  6. Add a new Email Service 
+     1. Currently only Gmail is supported. Stay tunned for instructions on how to add a new service.
```
Comment (evaldez89) [inline README.md:35]: don use specific sequence just `1. ` that way it renders automatically

1. blah
1. blah
   1. sub blah

File: README.md [hunk 3]
```diff
@@ -1,36 +1,47 @@
...
+
+:alarm_clock: Pending
 
 ## The current supported banks are:
-  - Banco BHD León (Dominican Republic)
-  - Banco Vimenca (Dominican Republic) `(in progress)`
+  - Banco BHD León (Dominican Republic) :white_check_mark:
+  - Banco Vimenca (Dominican Republic) :construction:
 
-## The current supported email services are:
```
Comment (evaldez89) [inline README.md:18]: This section is missing

**Review by evaldez89 — APPROVED** [review]

**Pull request merged**